src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -744,4 +748,63 @@ export class StudyViewPageStore {
             return Promise.resolve(data);
         }
     });
+
```

Comment:
fyi keep it in mind that we will need to adapt this to new SPA submission.  no change necessary now